package.json

```diff
@@ -18,15 +19,16 @@
     "yup": "^0.32.9"
   },
   "scripts": {
+    "start": "react-scripts start",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "format": "prettier --write .",
-    "formatHusky": "prettier --write",
-    "lint:fix": "eslint \"src/**/*.{js,jsx}\" --fix",
-    "prepare": "husky install",
-    "start": "react-scripts start",
     "test": "react-scripts test",
-    "testHusky": "react-scripts test --watchAll=false --passWithNoTests"
```

Comment:
I guess we need this for meanwhile, because we do not have tests on frontend yet

Reply:
for what? for the pipeline to pass?

Reply:
And for pre-commit to pass too

Reply:
fair enough, let me brought back the dead lines

Reply:
I restored the lines but Husky doesn't work for me locally, I'm getting a `not set as executable` error for husky bin
tried chmod+x but didn't work
could it be bc of the my dev container env? IDK, can somebody try and test locally this branch please?
![image](https://user-images.githubusercontent.com/85836692/127048211-36ec5df0-ecc8-4035-8860-13bb85af7239.png)

Reply:
got it fixed